File: add/codesnippet/CSharp/m-system.diagnostics.deb_8_2.cs
    switch (option1) {
        case MyOption.First:
           result = 1.0;
           break;
     
        // Insert additional cases.
     
        default:
           Debug.Fail("Unknown Option " + option1, "Result set to 1.0");
           result = 1.0;
           break;
     }

File: add/codesnippet/CPP/m-system.diagnostics.deb_8_2.cpp
       switch ( option )
       {
          case Option::First:
             result = 1.0;
             break;
    
          // Insert additional cases.
          
          default:
             #if defined(DEBUG)
             Debug::Fail( "Unknown Option" + option, "Result set to 1.0" );
             #endif
             result = 1.0;
             break;
       }